### PR TITLE
Markdown third parties 2024

### LINKED ISSUE #3598: Third Parties 2024
# Third Parties 2024

Third Parties illustration

If you're interested in contributing to the Third Parties chapter of the 2024 Web Almanac, please reply to this issue and indicate which role or roles best fit your interest and availability: [author](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Authors'-Guide), [reviewer](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Reviewers'-Guide), [analyst](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Analysts'-Guide), and/or [editor](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Editors'-Guide). You might be interested in exploring the changes to this year's version [here](https://github.com/HTTPArchive/almanac.httparchive.org/discussions/3619).

## Content team

| Lead | Authors | Reviewers | Analysts | Editors | Coordinator |
| ---- | ------- | --------- | -------- | ------- |    -------  |
| @zubairshafiq  | @zubairshafiq, @Yash-Vekaria, @ChrisBeeti   | @turban1988  | @Yash-Vekaria, @ChrisBeeti  | @neriiavr  | @turban1988   | 

<details>
<summary><b>Expand for more information about each role 👀</b></summary>

- The **[content team lead](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Content-Team-Leads'-Guide)** is the chapter owner and responsible for setting the scope of the chapter and managing contributors' day-to-day progress.
- **[Authors](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Authors'-Guide)** are subject matter experts and lead the content direction for each chapter. Chapters typically have one or two authors. Authors are responsible for planning the outline of the chapter, analyzing stats and trends, and writing the annual report.
- **[Reviewers](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Reviewers'-Guide)** are also subject matter experts and assist authors with technical reviews during the planning, analyzing, and writing phases.
- **[Analysts](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Analysts'-Guide)** are responsible for researching the stats and trends used throughout the Almanac. Analysts work closely with authors and reviewers during the planning phase to give direction on the types of stats that are possible from the dataset, and during the analyzing/writing phases to ensure that the stats are used correctly.
- **[Editors](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Editors'-Guide)** are technical writers who have a penchant for both technical and non-technical content correctness. Editors have a mastery of the English language and work closely with authors to help wordsmith content and ensure that everything fits together as a cohesive unit.
- The **[section coordinator](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Section-Leads'-Guide)** is the overall owner for all chapters within a section like "User Experience" or "Page Content" and helps to keep each chapter on schedule.

_Note: The time commitment for each role varies by the chapter's scope and complexity as well as the number of contributors._

For an overview of how the roles work together at each phase of the project, see the [Chapter Lifecycle](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Chapter-Lifecycle) doc.
</details>

### [0. Form the content team](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Chapter-Lifecycle#0-create-content-team)
- [x] 📆 `April 15` Complete program and content committee - 🔑 *Organizing committee*
  - The content team has at least one author, reviewer, and analyst.

### [1. Plan content](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Chapter-Lifecycle#1-plan-content)
- [x] 📆 `May 1` **First meeting to outline the chapter contents** - 🔑 *Content team*
  - The content team has completed the chapter outline.

### [2. Gather data](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Chapter-Lifecycle#2-gather-data)
- [x] 📆 `June 1` **Custom metrics completed** - 🔑 *Analysts*
  - Analysts have added all necessary [custom metrics](https://github.com/HTTPArchive/custom-metrics/blob/main/README.md) and drafted a PR ([example](https://github.com/HTTPArchive/almanac.httparchive.org/pull/1087)) to track query progress. 
- [x]  📆 `June 1` **HTTP Archive Crawl** - 🔑 *HA Team*
  - HTTP Archive runs the June crawl.

### [3. Validate results](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Chapter-Lifecycle#3-validate-results)
- [x] 📆 `August 15` **Query Metrics & Save Results** - 🔑 *Analysts*
  - Analysts have queried all metrics and saved the output.

### [4. Draft content](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Chapter-Lifecycle#4-draft-content)
- [x] 📆 `September 15` **First Draft of Chapter** - 🔑 *Authors*
  - Authors has written the chapter. 
- [x] 📆 `October 10` **Review & Edit Chapter** - 🔑 *Reviewers & Editors*
  - Reviewers and  Editors has processed the the chapter.

### [5. Publication](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Chapter-Lifecycle#5-publication)
- [x] 📆 `October 15` **Chapter Publication (Markdown & PR)** - 🔑 *Authors*
  - Authors has converted the chapter to markdown and drafted a PR.
- [ ] 📆 `November 1` **Launch of 2024 Web Almanac 🚀** - 🔑 *Organizing committee*
 
### [6. Virtual conference](#)
- [ ] 📆 `November 20` **Virtual Conference** - 🔑 *Content Team*

<a id="chapter-resources"></a>
## [Chapter resources](#chapter-resources)
Refer to these 2024 Third Parties resources throughout the content creation process:
[📄 Google Docs](https://docs.google.com/document/d/1BOJHpyGVOzQUUnAdDBxfjP6Lj1wwd6B3u830UVT8YBg/edit) for outlining and drafting content
[🔍 SQL files](https://github.com/HTTPArchive/almanac.httparchive.org/tree/main/sql/2024/third-parties/README.md) for committing the queries used during analysis
[📊 Google Sheets](https://docs.google.com/spreadsheets/d/18uTDBygSqgT_PNFldOz4guLSuXyMzDthRGnAG5if4sU/edit#gid=1778117656) for saving the results of queries
[📝 Markdown file](https://github.com/HTTPArchive/almanac.httparchive.org/tree/main/src/content/en/2023/third-parties.md) for publishing content and managing public metadata
[💻 Collab notebook](https://colab.research.google.com/drive/1bevj7ZX_8yZ3WH5w83QGVsmKIxTcyHLD) for collaborative coding in Python - if needed
[💬 #web-almanac-third-parties](https://join.slack.com/t/httparchive/shared_invite/zt-45sgwmnb-eDEatOhqssqNAKxxOSLAaA) on Slack for team coordination

### PR DESCRIPTION
**Pull request by ChrisBeeti**
Fixes #3598

Staged: https://third-parties-2024-dot-webalmanac.uk.r.appspot.com/en/2024/third-parties
Featured quote: https://third-parties-2024-dot-webalmanac.uk.r.appspot.com/en/2024/?feat=third-parties#featured-chapter

Third parties chapter MD file + png + SQL statement for gtm calls

### CODE REVIEW

File: src/content/en/2024/third-parties.md [hunk 1]
```diff
@@ -3,17 +3,205 @@
...
-authors: []
-reviewers: []
-editors: []
-analysts: []
+authors: [turban1988, Yash-Vekaria, zubairshafiq, ChrisBeeti]
+reviewers: [tunetheweb]
+editors: [neriiavr]
+analysts: [Yash-Vekaria, ChrisBeeti]
 translators: []
+turban1988_bio: TODO
```
Comment (tunetheweb) [inline src/content/en/2024/third-parties.md:11]: @turban1988 can you please provide a short bio about yourself?

File: src/content/en/2024/third-parties.md [hunk 2]
```diff
@@ -3,17 +3,205 @@
...
+Yash-Vekaria_bio: TODO
```
Comment (tunetheweb) [inline src/content/en/2024/third-parties.md:12]: @Yash-Vekaria can you please provide a short bio about yourself?

File: src/content/en/2024/third-parties.md [hunk 3]
```diff
@@ -3,17 +3,205 @@
...
+zubairshafiq_bio: TODO
```
Comment (tunetheweb) [inline src/content/en/2024/third-parties.md:13]: @zubairshafiq can you please provide a short bio about yourself?

File: src/content/en/2024/third-parties.md [hunk 4]
```diff
@@ -3,17 +3,205 @@
...
+ChrisBeeti_bio: TODO
```
Comment (tunetheweb) [inline src/content/en/2024/third-parties.md:14]: @ChrisBeeti can you please provide a short bio about yourself?

File: src/content/en/2024/third-parties.md [hunk 5]
```diff
@@ -3,17 +3,205 @@
...
-featured_stat_3:
-featured_stat_label_3:
+featured_quote: Google is the most popular third-parties on the web, with five of the top ten third-party domains being Google domains.
+featured_stat_1: 92%
+featured_stat_label_1: Web pages including one or more third parties
+featured_stat_2: 78
+featured_stat_label_2: The median number of third-party requests per web page
+featured_stat_3: 22
+featured_stat_label_3: The median number of third-parties per web page
+doi: TODO
```
Comment (tunetheweb) [inline src/content/en/2024/third-parties.md:23]: @nrllh can you please provide the doi for this chapter?

File: src/content/en/2024/third-parties.md (hunk 4 above)
Comment (ChrisBeeti) [reply src/content/en/2024/third-parties.md:14]:
```suggestion
ChrisBeeti_bio: Chris Böttger is a PhD candidate in Computer Science at the [Westphalian University of Applied Science](https://www.en.w-hs.de/). His research focuses on web and network security, primarily focusing on user privacy and tracking technologies.
```

File: src/content/en/2024/third-parties.md (hunk 1 above)
Comment (ChrisBeeti) [reply src/content/en/2024/third-parties.md:11]:
```suggestion
turban1988_bio: Prof. Dr.-Ing. Tobias Urban, Professor of Computer Science, specializing in Cybersecurity, at the [Westphalian University of Applied Sciences](https://www.en.w-hs.de/). His research focuses on IT security, data protection, and the impact of the GDPR on the Internet.
```

File: src/content/en/2024/third-parties.md (hunk 2 above)
Comment (Yash-Vekaria) [reply src/content/en/2024/third-parties.md:12]: Yash Vekaria is a PhD candidate in Computer Science at [University of California, Davis](https://www.ucdavis.edu). He carries out web-based large-scale Internet measurements to study and improve the dynamics of web. Specifically, his research is focused at studying and bringing transparency to online tracking practices and user privacy issues.

File: src/content/en/2024/third-parties.md (hunk 5 above)
Comment (nrllh) [reply src/content/en/2024/third-parties.md:23]: Sure, here: `10.5281/zenodo.14193383`

File: src/content/en/2024/third-parties.md (hunk 2 above)
Comment (tunetheweb) [reply src/content/en/2024/third-parties.md:12]:
```suggestion
Yash-Vekaria_bio: Yash Vekaria is a PhD candidate in Computer Science at <a hreflang="en" href="https://www.ucdavis.edu/">University of California, Davis</a>. He carries out web-based large-scale Internet measurements to study and improve the dynamics of web. Specifically, his research is focused at studying and bringing transparency to online tracking practices and user privacy issues.
```

File: src/content/en/2024/third-parties.md (hunk 5 above)
Comment (nrllh) [reply src/content/en/2024/third-parties.md]: @tunetheweb just updated the DOI

File: src/content/en/2024/third-parties.md (hunk 3 above)
Comment (Yash-Vekaria) [reply src/content/en/2024/third-parties.md:13]: @tunetheweb Please use this:
Zubair Shafiq is a computer science professor at [University of California, Davis](https://www.ucdavis.edu/). His research aims to make the Internet private, secure, and safe using empirically grounded measurement and modeling methods.

Comment (tunetheweb) [reply src/content/en/2024/third-parties.md]: Done!

**Review by tunetheweb — APPROVED** [review]

**Pull request merged**